### PR TITLE
PS-7918 Add and update documents for Netlify conversion (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   Makefile
new file:   netlify.toml
new file:   requirements.txt
new file:   runtime.txt
modified:   source/changed_in_57.rst
new file:   source/conf-netlify/conf.py
modified:   source/development.rst
modified:   source/diagnostics/innodb_fragmentation_count.rst
modified:   source/diagnostics/innodb_show_status.rst
modified:   source/diagnostics/libcoredumper.rst
modified:   source/diagnostics/misc_info_schema_tables.rst
modified:   source/diagnostics/process_list.rst
modified:   source/diagnostics/response_time_distribution.rst
modified:   source/diagnostics/scalability_metrics_plugin.rst
modified:   source/diagnostics/show_engines.rst
modified:   source/diagnostics/slow_extended.rst
modified:   source/diagnostics/stacktrace.rst
modified:   source/diagnostics/thread_based_profiling.rst
modified:   source/diagnostics/user_stats.rst
modified:   source/faq.rst
modified:   source/feature_comparison.rst
modified:   source/flexibility/binlogging_replication_improvements.rst
modified:   source/flexibility/compressed_columns.rst
modified:   source/flexibility/csv_engine_mode.rst
modified:   source/flexibility/extended_mysqlbinlog.rst
modified:   source/flexibility/extended_mysqldump.rst
modified:   source/flexibility/extended_select_into_outfile.rst
modified:   source/flexibility/improved_memory_engine.rst
modified:   source/flexibility/innodb_fts_improvements.rst
modified:   source/flexibility/log_warnings_suppress.rst
modified:   source/flexibility/max_binlog_files.rst
modified:   source/flexibility/per_query_variable_statement.rst
modified:   source/flexibility/per_session_server-id.rst
modified:   source/flexibility/proxy_protocol_support.rst
modified:   source/flexibility/slowlog_rotation.rst
modified:   source/glossary.rst
modified:   source/index.rst
modified:   source/index_info_schema_tables.rst
modified:   source/installation.rst
modified:   source/installation/apt_repo.rst
modified:   source/installation/post-installation.rst
modified:   source/installation/yum_repo.rst
modified:   source/management/audit_log_plugin.rst
modified:   source/management/backup_locks.rst
modified:   source/management/changed_page_tracking.rst
modified:   source/management/enforce_engine.rst
modified:   source/management/expanded_program_option_modifiers.rst
modified:   source/management/extended_show_grants.rst
modified:   source/management/innodb_expanded_fast_index_creation.rst
modified:   source/management/innodb_kill_idle_trx.rst
modified:   source/management/start_transaction_with_consistent_snapshot.rst
modified:   source/management/udf_percona_toolkit.rst
modified:   source/management/utility_user.rst
modified:   source/myrocks/gap_locks_detection.rst
modified:   source/myrocks/information-schema-tables.rst
modified:   source/myrocks/status_variables.rst
modified:   source/myrocks/variables.rst
modified:   source/percona_xtradb.rst
modified:   source/performance/aio_page_requests.rst
modified:   source/performance/innodb_numa_support.rst
modified:   source/performance/prefix_index_queries_optimization.rst
modified:   source/performance/threadpool.rst
modified:   source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst
modified:   source/ps-versions-comparison.rst
modified:   source/release-notes/Percona-Server-5.7.10-1.rst
modified:   source/release-notes/Percona-Server-5.7.10-2.rst
modified:   source/release-notes/Percona-Server-5.7.10-3.rst
modified:   source/release-notes/Percona-Server-5.7.11-4.rst
modified:   source/release-notes/Percona-Server-5.7.12-5.rst
modified:   source/release-notes/Percona-Server-5.7.13-6.rst
modified:   source/release-notes/Percona-Server-5.7.14-7.rst
modified:   source/release-notes/Percona-Server-5.7.14-8.rst
modified:   source/release-notes/Percona-Server-5.7.15-9.rst
modified:   source/release-notes/Percona-Server-5.7.16-10.rst
modified:   source/release-notes/Percona-Server-5.7.17-11.rst
modified:   source/release-notes/Percona-Server-5.7.17-12.rst
modified:   source/release-notes/Percona-Server-5.7.17-13.rst
modified:   source/release-notes/Percona-Server-5.7.18-14.rst
modified:   source/release-notes/Percona-Server-5.7.18-15.rst
modified:   source/release-notes/Percona-Server-5.7.18-16.rst
modified:   source/release-notes/Percona-Server-5.7.20-18.rst
modified:   source/release-notes/Percona-Server-5.7.20-19.rst
modified:   source/release-notes/Percona-Server-5.7.21-20.rst
modified:   source/release-notes/Percona-Server-5.7.21-21.rst
modified:   source/release-notes/Percona-Server-5.7.22-22.rst
modified:   source/release-notes/Percona-Server-5.7.23-23.rst
modified:   source/release-notes/Percona-Server-5.7.24-26.rst
modified:   source/release-notes/Percona-Server-5.7.24-27.rst
modified:   source/release-notes/Percona-Server-5.7.25-28.rst
modified:   source/release-notes/Percona-Server-5.7.26-29.rst
modified:   source/release-notes/Percona-Server-5.7.27-30.rst
modified:   source/release-notes/Percona-Server-5.7.28-31.rst
modified:   source/release-notes/release-notes_index.rst
modified:   source/reliability/innodb_corrupt_table_action.rst
modified:   source/reliability/log_connection_error.rst
modified:   source/scalability/innodb_io.rst
modified:   source/scalability/innodb_split_buf_pool_mutex.rst
modified:   source/security/data-at-rest-encryption.rst
modified:   source/security/data-masking.rst
modified:   source/security/pam_plugin.rst
modified:   source/security/ssl-improvement.rst
modified:   source/tokudb/fast_updates.rst
modified:   source/tokudb/removing_tokudb.rst
modified:   source/tokudb/toku_backup.rst
modified:   source/tokudb/tokudb_background_analyze_table.rst
modified:   source/tokudb/tokudb_faq.rst
modified:   source/tokudb/tokudb_file_management.rst
modified:   source/tokudb/tokudb_files_and_file_types.rst
modified:   source/tokudb/tokudb_installation.rst
modified:   source/tokudb/tokudb_intro.rst
modified:   source/tokudb/tokudb_performance_schema.rst
modified:   source/tokudb/tokudb_quickstart.rst
modified:   source/tokudb/tokudb_status_variables.rst
modified:   source/tokudb/tokudb_troubleshooting.rst
modified:   source/tokudb/tokudb_variables.rst
modified:   source/tokudb/using_tokudb.rst
modified:   source/upgrading_guide_56_57.rst
modified:   source/upstream-bug-fixes.rst

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (nastena1606) [inline doc/requirements.txt:1]: You don't have Sphinx itself in the list

File: doc/runtime.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+3.7
```
Comment (nastena1606) [inline doc/runtime.txt]: You may need to change it to 3.8. Needs to be tested

File: doc/requirements.txt (hunk 1 above)
Comment (nastena1606) [reply doc/requirements.txt:1]: Please add sphinx-gitstamp==0.3.1 as you have this extension included in netlify/conf.py

File: doc/source/installation.rst [hunk 3]
```diff
@@ -1,15 +1,15 @@
 .. _installation:
 
 ===============================
-Installing |Percona Server| 5.7
+Installing Percona Server for MySQL 5.7
```
Comment (nastena1606) [inline doc/source/installation.rst:4]: I think the strokes should be longer

File: doc/source/installation.rst [hunk 4]
```diff
@@ -20,10 +20,10 @@ Before installing, you might want to read the :doc:`release-notes/release-notes_
 
 .. _installing_from_binaries:
 
-Installing |Percona Server| from Repositories
+Installing Percona Server for MySQL from Repositories
 =============================================
```
Comment (nastena1606) [inline doc/source/installation.rst]: I think the strokes should be longer

File: doc/source/installation.rst [hunk 5]
```diff
@@ -36,10 +36,10 @@ Following guides describe the installation process for using the official Percon
 
 .. _installing_from_binary_tarball:
 
-Installing |Percona Server| from a Binary Tarball
+Installing Percona Server for MySQL from a Binary Tarball
```
Comment (nastena1606) [inline doc/source/installation.rst:39]: I think the strokes should be longer

File: doc/source/installation.rst [hunk 6]
```diff
@@ -76,7 +76,7 @@ Fetch and extract the correct binary tarball. For example for *Debian Wheezy*:
 
 .. _installing_from_source_tarball:
 
-Installing |Percona Server| from a Source Tarball
+Installing Percona Server for MySQL from a Source Tarball
```
Comment (nastena1606) [inline doc/source/installation.rst:79]: I think the strokes should be longer

File: doc/source/installation.rst [hunk 7]
```diff
@@ -116,15 +116,15 @@ Next, follow the instructions in :ref:`compile_from_source` below.
 
 .. _compile_from_source:
 
-Compiling |Percona Server| from Source
+Compiling Percona Server for MySQL from Source
```
Comment (nastena1606) [inline doc/source/installation.rst:119]: I think the strokes should be longer

File: doc/source/installation.rst [hunk 8]
```diff
@@ -142,7 +142,7 @@ Install: ::
 
 Percona Server 5.7 will now be installed on your system.
 
-Building |Percona Server| Debian/Ubuntu packages
+Building Percona Server for MySQL Debian/Ubuntu packages
```
Comment (nastena1606) [inline doc/source/installation.rst:145]: I think the strokes should be longer

File: doc/source/installation/apt_repo.rst [hunk 9]
```diff
@@ -1,10 +1,10 @@
 .. _apt_repo:
 
 ====================================================
-Installing |Percona Server| on *Debian* and *Ubuntu*
+Installing Percona Server for MySQL on *Debian* and *Ubuntu*
```
Comment (nastena1606) [inline doc/source/installation/apt_repo.rst:4]: I think the strokes should be longer

File: doc/source/installation/apt_repo.rst [hunk 10]
```diff
@@ -27,7 +27,7 @@ The ``libperconaserverclient20-dev`` package contains header files needed to com
 
 The ``libperconaserverclient20`` package contains the client shared library. The ``18.1`` is a reference to the version of the shared library. The version is incremented when there is a ABI change that requires software using the client library to be recompiled or its source code modified.
 
-Installing |Percona Server| from Percona ``apt`` repository
+Installing Percona Server for MySQL from Percona ``apt`` repository
```
Comment (nastena1606) [inline doc/source/installation/apt_repo.rst:30]: I think the strokes should be longer

File: doc/source/installation/apt_repo.rst [hunk 11]
```diff
@@ -105,10 +105,10 @@ For more information about the pinning you can check the official `debian wiki <
 
 .. _standalone_deb:
 
-Installing |Percona Server| using downloaded deb packages
+Installing Percona Server for MySQL using downloaded deb packages
```
Comment (nastena1606) [inline doc/source/installation/apt_repo.rst:108]: I think the strokes should be longer

File: doc/source/installation/apt_repo.rst [hunk 12]
```diff
@@ -173,17 +173,17 @@ The following table lists the default locations for files:
 
   *Debian* and *Ubuntu* installation does not automatically create a special
   ``debian-sys-maint`` user which can be used by the control scripts to control
-  the |Percona Server| ``mysqld`` and ``mysqld_safe`` services like it was the
-  case with previous |Percona Server| versions. If you still require this user you must create the user manually.
+  the Percona Server for MySQL ``mysqld`` and ``mysqld_safe`` services like it was the
+  case with previous Percona Server for MySQL versions. If you still require this user you must create the user manually.
 
-Running |Percona Server|
+Running Percona Server for MySQL
```
Comment (nastena1606) [inline doc/source/installation/apt_repo.rst:179]: I think the strokes should be longer

File: doc/source/installation/apt_repo.rst [hunk 13]
```diff
@@ -219,10 +219,10 @@ The following procedure runs the |Percona Server|:
 
   *Debian* 8.0 (jessie) and *Ubuntu* 16.04(Xenial) come with `systemd <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and service manager so you can invoke all the above commands with ``sytemctl`` instead of ``service``. Currently, both are supported.
 
-Uninstalling |Percona Server|
+Uninstalling Percona Server for MySQL
 =============================
```
Comment (nastena1606) [inline doc/source/installation/apt_repo.rst]: I think the strokes should be longer

File: doc/source/installation/yum_repo.rst [hunk 14]
```diff
@@ -1,10 +1,10 @@
 .. _yum_repo:
 
 ====================================================================
- Installing |Percona Server| on Red Hat Enterprise Linux and CentOS
+ Installing Percona Server for MySQL on Red Hat Enterprise Linux and CentOS
```
Comment (nastena1606) [inline doc/source/installation/yum_repo.rst:4]: I think the strokes should be longer

File: doc/source/installation/yum_repo.rst [hunk 15]
```diff
@@ -27,9 +27,9 @@ The ``Percona-Server-shared-57`` package includes the client shared library.
 
 The ``Percona-Server-shared-compat`` package includes shared libraries for software compiled against old versions of the client library. Following libraries are included in this package: ``libmysqlclient.so.12``, ``libmysqlclient.so.14``, ``libmysqlclient.so.15``, ``libmysqlclient.so.16``, and ``libmysqlclient.so.18``.
 
-The ``Percona-Server-test-57`` package includes the test suite for |Percona Server|.
+The ``Percona-Server-test-57`` package includes the test suite for Percona Server for MySQL.
 
-Installing |Percona Server| from Percona ``yum`` repository
+Installing Percona Server for MySQL from Percona ``yum`` repository
```
Comment (nastena1606) [inline doc/source/installation/yum_repo.rst:32]: I think the strokes should be longer

File: doc/source/installation/yum_repo.rst [hunk 16]
```diff
@@ -106,10 +106,10 @@ Percona offers pre-release builds from our testing repository. To subscribe to t
 
 .. _standalone_rpm:
 
-Installing |Percona Server| using downloaded rpm packages
+Installing Percona Server for MySQL using downloaded rpm packages
```
Comment (nastena1606) [inline doc/source/installation/yum_repo.rst:109]: I think the strokes should be longer

File: doc/source/installation/yum_repo.rst [hunk 17]
```diff
@@ -190,12 +190,12 @@ You can use the following command to locate the Data directory:
     datadir=/var/lib/mysql
 
 
-Running |Percona Server|
+Running Percona Server for MySQL
```
Comment (nastena1606) [inline doc/source/installation/yum_repo.rst:193]: I think the strokes should be longer

File: doc/source/installation/yum_repo.rst [hunk 18]
```diff
@@ -230,12 +230,12 @@ Running |Percona Server|
 
   *RHEL* 7 and *CentOS* 7 come with `systemd <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and service manager so you can invoke all the above commands with ``sytemctl`` instead of ``service``. Currently both are supported.
 
-Uninstalling |Percona Server|
+Uninstalling Percona Server for MySQL
```
Comment (nastena1606) [inline doc/source/installation/yum_repo.rst:233]: I think the strokes should be longer

File: doc/source/ps-versions-comparison.rst [hunk 19]
```diff
@@ -1,17 +1,17 @@
 .. _ps_versions_comparison:
 
 =======================================================
-List of features available in |Percona Server| releases
+List of features available in Percona Server for MySQL releases
```
Comment (nastena1606) [inline doc/source/ps-versions-comparison.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.10-1.rst [hunk 20]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.10-1
 
 =========================
-|Percona Server| 5.7.10-1
+Percona Server for MySQL 5.7.10-1
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.10-1.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.10-2.rst [hunk 21]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.10-2
 
 =========================
-|Percona Server| 5.7.10-2
+Percona Server for MySQL 5.7.10-2
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.10-2.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.10-3.rst [hunk 22]
```diff
@@ -1,40 +1,40 @@
 .. rn:: 5.7.10-3
 
 =========================
-|Percona Server| 5.7.10-3
+Percona Server for MySQL 5.7.10-3
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.10-3.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.11-4.rst [hunk 23]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.11-4
 
 =========================
-|Percona Server| 5.7.11-4
+Percona Server for MySQL 5.7.11-4
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.11-4.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.12-5.rst [hunk 24]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.12-5
 
 ===========================
- |Percona Server| 5.7.12-5
+ Percona Server for MySQL 5.7.12-5
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.12-5.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.13-6.rst [hunk 25]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.13-6
 
 ===========================
- |Percona Server| 5.7.13-6
+ Percona Server for MySQL 5.7.13-6
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.13-6.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.14-7.rst [hunk 26]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.14-7
 
 ===========================
- |Percona Server| 5.7.14-7
+ Percona Server for MySQL 5.7.14-7
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.14-7.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.14-8.rst [hunk 27]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.14-8
 
 ===========================
- |Percona Server| 5.7.14-8
+ Percona Server for MySQL 5.7.14-8
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.14-8.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.15-9.rst [hunk 28]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.15-9
 
 =========================
-|Percona Server| 5.7.15-9
+Percona Server for MySQL 5.7.15-9
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.15-9.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.16-10.rst [hunk 29]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.16-10
 
 ==========================
-|Percona Server| 5.7.16-10
+Percona Server for MySQL 5.7.16-10
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.16-10.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.17-11.rst [hunk 30]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.17-11
 
 ==========================
-|Percona Server| 5.7.17-11
+Percona Server for MySQL 5.7.17-11
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.17-11.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst [hunk 31]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.17-12
 
 ==========================
-|Percona Server| 5.7.17-12
+Percona Server for MySQL 5.7.17-12
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.17-12.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.17-13.rst [hunk 32]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.17-13
 
 ==========================
-|Percona Server| 5.7.17-13
+Percona Server for MySQL 5.7.17-13
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.17-13.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.18-14.rst [hunk 33]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.18-14
 
 ==========================
-|Percona Server| 5.7.18-14
+Percona Server for MySQL 5.7.18-14
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.18-14.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.18-15.rst [hunk 34]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.18-15
 
 ==========================
-|Percona Server| 5.7.18-15
+Percona Server for MySQL 5.7.18-15
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.18-15.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst [hunk 35]
```diff
@@ -1,7 +1,7 @@
 .. rn:: 5.7.18-16
 
 ==========================
-|Percona Server| 5.7.18-16
+Percona Server for MySQL 5.7.18-16
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.18-16.rst:4]: I think the strokes should be longer

File: doc/source/release-notes/Percona-Server-5.7.24-27.rst [hunk 36]
```diff
@@ -1,35 +1,35 @@
 .. rn:: 5.7.24-27
 	
 ==========================
-|Percona Server| |release|
+Percona Server for MySQL 5.7.24-27
```
Comment (nastena1606) [inline doc/source/release-notes/Percona-Server-5.7.24-27.rst:4]: I think the strokes should be longer

File: doc/requirements.txt (hunk 1 above)
Comment (patrickbirch) [reply doc/requirements.txt:1]: Added.

File: doc/source/installation.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/installation.rst:4]: Fixed.

File: doc/source/installation.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/installation.rst:39]: Fixed.

File: doc/source/installation.rst (hunk 7 above)
Comment (patrickbirch) [reply doc/source/installation.rst:119]: Fixed.

File: doc/source/installation/apt_repo.rst (hunk 9 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst:4]: Fixed.

File: doc/source/installation/apt_repo.rst (hunk 10 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst:30]: Fixed.

File: doc/source/installation/apt_repo.rst (hunk 12 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst:179]: Fixed.

File: doc/source/installation/apt_repo.rst (hunk 13 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst]: Fixed.

File: doc/source/installation/yum_repo.rst (hunk 15 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst:32]: Fixed.

File: doc/source/installation/yum_repo.rst (hunk 16 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst:109]: Fixed.

File: doc/source/installation/yum_repo.rst (hunk 18 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst:233]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.10-1.rst (hunk 20 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.10-1.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.10-2.rst (hunk 21 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.10-2.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.10-3.rst (hunk 22 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.10-3.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.11-4.rst (hunk 23 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.11-4.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.12-5.rst (hunk 24 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.12-5.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.13-6.rst (hunk 25 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.13-6.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.14-7.rst (hunk 26 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.14-7.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.14-8.rst (hunk 27 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.14-8.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.15-9.rst (hunk 28 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.15-9.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.16-10.rst (hunk 29 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.16-10.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.17-11.rst (hunk 30 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.17-11.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst (hunk 31 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.17-12.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.17-13.rst (hunk 32 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.17-13.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.18-14.rst (hunk 33 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.18-14.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.18-15.rst (hunk 34 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.18-15.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.18-16.rst (hunk 35 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.18-16.rst:4]: Fixed.

File: doc/source/release-notes/Percona-Server-5.7.24-27.rst (hunk 36 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.24-27.rst:4]: Fixed.

File: doc/source/installation/yum_repo.rst (hunk 17 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst:193]: Fixed.

File: doc/source/installation/yum_repo.rst (hunk 14 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst:4]: Fixed.

File: doc/source/installation/apt_repo.rst (hunk 11 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst:108]: Fixed.

File: doc/source/installation.rst (hunk 8 above)
Comment (patrickbirch) [reply doc/source/installation.rst:145]: Fixed.

File: doc/source/installation.rst (hunk 6 above)
Comment (patrickbirch) [reply doc/source/installation.rst:79]: Fixed.

File: doc/source/installation.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/installation.rst]: Fixed.

File: doc/runtime.txt (hunk 2 above)
Comment (patrickbirch) [reply doc/runtime.txt]: Updated.

File: doc/requirements.txt (hunk 1 above)
Comment (patrickbirch) [reply doc/requirements.txt:1]: Updated.

**Comment by patrickbirch** [comment]
No longer using Netlify

**Pull request closed without merging**